services/authentication/src/main/java/entity/User.java

```diff
@@ -0,0 +1,35 @@
+package entity;
+
+/**
+ * Created by Michele on 12/07/2017.
+ */
+public class User {
+    private String username;
+    private String password;
+
+    public User(String username, String password) {
+        this.username = username;
+        this.password = password;
+    }
+
+    public User() {
+    }
+
+    public String getUsername() {
+        return this.username;
+    }
+
+    public void setUsername(String username) {
```

Comment:
I think that the setters can be usefull in case we want to implement, in the future, an  user's username/password change mechanism.